cf/app_files/app_files.go

```diff
@@ -139,9 +139,18 @@ func copyPathToPath(fromPath, toPath string) (err error) {
 
 func loadIgnoreFile(dir string) CfIgnore {
 	fileContents, err := ioutil.ReadFile(filepath.Join(dir, ".cfignore"))
+
 	if err == nil {
 		return NewCfIgnore(string(fileContents))
 	} else {
-		return NewCfIgnore("")
+		// FIXME: Should we extend to other VCS systems beyond .gitignore
```

Comment:
I don't really think this is a "FIXME" since it would be a new "feature" but no biggie.